ipld/plugin/nmt_test.go

```diff
@@ -41,3 +44,27 @@ func TestNamespaceFromCID(t *testing.T) {
 		})
 	}
 }
+
+// generateRandNamespacedRawData returns random namespaced raw data for testing purposes.
+func generateRandNamespacedRawData(total, nidSize, leafSize uint32) [][]byte {
```

Comment:
This helper exists now 4 times in tests. I don't really mind (as it's only in tests but maybe this should actually again be moved into a testutil package or export it only once; in a separate PR though IMHO).